cicecore/cicedyn/analysis/ice_history_pond.F90

```diff
@@ -456,58 +507,36 @@ subroutine accum_hist_pond (iblk)
-                                                   * trcr(:,:,nt_hpnd,iblk), a2D)
+             call accum_hist_field(n_hpond, iblk, worka(:,:)*trcr(:,:,nt_hpnd,iblk), a2D)
          if (f_hpond_ai(1:1)/= 'x') &
-             call accum_hist_field(n_hpond_ai, iblk, &
-                            aice(:,:,iblk) &
-                          * trcr(:,:,nt_alvl,iblk) * trcr(:,:,nt_apnd,iblk) &
-                                                   * trcr(:,:,nt_hpnd,iblk), a2D)
+             call accum_hist_field(n_hpond_ai, iblk, aice_init(:,:,iblk)*worka(:,:)*trcr(:,:,nt_hpnd,iblk), a2D)
          if (f_ipond(1:1)/= 'x') &
-             call accum_hist_field(n_ipond, iblk, &
-                            trcr(:,:,nt_alvl,iblk) * trcr(:,:,nt_apnd,iblk) &
-                                                   * trcr(:,:,nt_ipnd,iblk), a2D)
+             call accum_hist_field(n_ipond, iblk, worka(:,:)*trcr(:,:,nt_ipnd,iblk), a2D)
          if (f_ipond_ai(1:1)/= 'x') &
-             call accum_hist_field(n_ipond_ai, iblk, &
-                            aice(:,:,iblk) &
-                          * trcr(:,:,nt_alvl,iblk) * trcr(:,:,nt_apnd,iblk) &
-                                                   * trcr(:,:,nt_ipnd,iblk), a2D)
+             call accum_hist_field(n_ipond_ai, iblk, aice_init(:,:,iblk)*worka(:,:)*trcr(:,:,nt_ipnd,iblk), a2D)
 
-         elseif (tr_pond_topo .or. tr_pond_sealvl) then
+         ! CMIP pond related variables
+         if (f_simpeffconc (1:1) /= 'x') &
+             call accum_hist_field(n_simpeffconc, iblk, apeff_ai(:,:,iblk), a2D)
 
-         if (f_apond(1:1)/= 'x') &
-             call accum_hist_field(n_apond, iblk, &
-                                   trcr(:,:,nt_apnd,iblk), a2D)
-         if (f_apond_ai(1:1)/= 'x') &
-             call accum_hist_field(n_apond_ai, iblk, &
-                                   aice(:,:,iblk) * trcr(:,:,nt_apnd,iblk), a2D)
-         if (f_hpond(1:1)/= 'x') &
-             call accum_hist_field(n_hpond, iblk, &
-                                   trcr(:,:,nt_apnd,iblk) &
-                                 * trcr(:,:,nt_hpnd,iblk), a2D)
-         if (f_hpond_ai(1:1)/= 'x') &
-             call accum_hist_field(n_hpond_ai, iblk, &
-                                   aice(:,:,iblk) * trcr(:,:,nt_apnd,iblk) &
-                                                  * trcr(:,:,nt_hpnd,iblk), a2D)
-         if (f_ipond(1:1)/= 'x') &
-             call accum_hist_field(n_ipond, iblk, &
-                                   trcr(:,:,nt_apnd,iblk) &
-                                 * trcr(:,:,nt_ipnd,iblk), a2D)
-         if (f_ipond_ai(1:1)/= 'x') &
-             call accum_hist_field(n_ipond_ai, iblk, &
-                                   aice(:,:,iblk) * trcr(:,:,nt_apnd,iblk) &
-                                                  * trcr(:,:,nt_ipnd,iblk), a2D)
-         endif ! ponds
+         if (f_simpconc(1:1)/= 'x') &
+             call accum_hist_field(n_simpconc, iblk, worka(:,:), a2D)
```

Comment:
Is simpconc divided by sum(ice area) later, along with all the other variables?  If so, then ice area should be multiplied here.  That would make it consistent with all the other variables too.

Reply:
So `avg_ice_present = 'none'`?